cockroachcloud/use-managed-service-backups.md

```diff
@@ -125,6 +124,20 @@ Additional ways to restore data:
 - [Back up a self-hosted CockroachDB cluster and restore into a {{ site.data.products.db }} cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachdb-cloud-cluster)
 - [Back up and restore data manually](take-and-restore-customer-owned-backups.html)
 
+### Restore a cluster
+
+To restore a cluster:
+
+1. Find the cluster backup on the **Backups** page.
+1. Click **Restore** for the cluster you want to restore.
+
+    The **Restore cluster** module displays with backup details.
+
+1. Click **Continue**
+1. Once you have reviewed the restore details, click **Restore**.
+
+   The **Restore Jobs** tab will show you the status of your restore and update when the restore job has been created successfully.
```

Comment:
Add a warning? This will wipe all data currently in the database to replace it with the restore. This is obvious, and we say so in the GUI, but someone will still miss this and the more documentation when that happens the better. Thanks!

Reply:
Good call! Better to be explicit about it!